js/test/unit/decoder/covtDecoder.spec.ts

```diff
@@ -85,19 +103,25 @@ function mapMvtProperties(mvtProperties: Map<string, unknown>): Map<string, unkn
     return transformedMvtProperties;
 }
 
-function getTiles(dir: string): { mvt: string; mlt: string }[] {
-    const tiles = fs.readdirSync(dir)
+function getTiles(dir: string): { mvt: string; mlt: string; mltMetadata: string }[] {
+    return fs.readdirSync(dir)
         .filter(file => file.endsWith('.mvt') || file.endsWith('.pbf'))
-        .sort();
-    const mappedTiles = [];
-    for (let i = 0; i < tiles.length; i += 2) {
-        const tile = tiles[i];
-        const mvtPath = Path.join(dir, tile);
-        const mltPath = Path.join(dir.replace('fixtures','expected'), tile.replace(/\.(pbf|mvt)$/,'.mlt'));
-        const tilePair = { mlt: mltPath, mvt: mvtPath };
-        mappedTiles.push(tilePair);
-    }
-    return mappedTiles;
+        .map((mvtFilename) : { mvt: string; mlt: string; mltMetadata: string } => {
+            const mvtPath = Path.join(dir, mvtFilename);
+
+            const mltDir = dir.replace('fixtures', 'expected');
+            const mltFilename = mvtFilename.replace(/\.(pbf|mvt)$/,'.mlt');
+            const mltPath = Path.join(mltDir, mltFilename);
+
+            const mltMetadataFilename = mltFilename + '.meta.pbf';
+            const mltMetadataPath = Path.join(mltDir, mltMetadataFilename);
```

Comment:
This works, but a high level thought is that the meta.pbf will have, if I'm understanding the metadata format correctly, a 1:N relationship with a set of tiles. So I think the ideal path structure to honestly represent that, and avoid having to store duplicate `.meta.pbf` would be:

> test/fixtures/<tilesource>/meta.pbf

Where `tilesource` is `bing` or `stamen`, etc.

I think, if I'm reading this code right, it is doing `test/fixtures/<tilesource>/z-x-y.meta.pbf`. But for every `z-x-y` the meta data should be the same, right? If not, then of course I'll need to dig into/understand why and this comment will need revisited.

Reply:
Yeah that's a good point, I was originally thinking of smaller fixtures that were likely to all have different metadata, but in this case it's definitely redundant. I'll make that change.